File: app/utils/DBObjectUtil.java
package utils;

import com.mongodb.BasicDBObject;

import java.util.ArrayList;
import java.util.List;

public class DBObjectUtil {


      public static List<Object> toList(BasicDBObject dbObject) {

          List<Object> values = new ArrayList<Object>();

          for(Object value : dbObject.values()){
            values.add(value);
          }

          return values;
      }

}

File: app/utils/DBObjectUtil.scala
package utils

import com.mongodb.BasicDBObject

import java.util.List

object DBObjectUtil {


     def toList(dbObject: BasicDBObject ):List[Any] = {

          val values = List()



          for(value <- dbObject.values()){
            values.add(value)
          }

          values
      }

}
